packages/lab/stories/progress/progress.doc.stories.mdx

```diff
@@ -0,0 +1,112 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { CircularProgress, LinearProgress } from "@salt-ds/lab";
+import {
+  CharacteristicUsage,
+  CSSClassTable,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Lab/Progress"
+  component={CircularProgress}
```

Comment:
I wonder if we should split this out into Documentation/Lab/Progress/Linear and Documentation/Lab/Progress/Circular to follow how the stories are, and also fix this being one component of two

Reply:
yeah I thought about this, not sure about how to structure it but I can take a look